vta/hardware/chisel/src/main/scala/dpi/VTAHostDPI.scala

```diff
@@ -70,3 +73,83 @@ class VTAHostDPI extends BlackBox with HasBlackBoxResource {
   })
   setResource("/verilog/VTAHostDPI.v")
 }
+
+/** Host DPI to AXI Converter.
+  *
+  * Convert Host DPI to AXI for VTAShell
```

Comment:
Would be good to provide pointers to info about DPI and AXI.

Reply:
You mean like urls? or short explanations?

DPI is the "CFFI" of Verilog (what we used for TSIM)
AXI is a bus protocol normally found on Xilinx FPGAs or ARM SoCs.

Reply:
Yeah, like places to read more, again thinking about people like me who don't have as much background but want to understand the code.

Reply:
I don't know how to proceed on this one since there are different files that contain DPI and AXI. I think a quick google search will help.